lib/src/page_list_viewport_gestures.dart

```diff
@@ -178,9 +181,13 @@ class _PageListViewportGesturesState extends State<PageListViewportGestures> wit
       return;
     }
 
-    if (translation.distance < 0.000001) {
-      // This means a distance of zero. We can't make a decision with zero translation.
-      // Fizzle and wait for the next panning notification.
+    if (translation.distance < KViewportScaleThresholds.minAxisLockingTranslationDistance) {
+      // The translation distance is not sufficiently large to be
+      // considered. Small translations should cause panning. This also
```

Comment:
Did you mean to say "Small translations should not cause panning"?

Reply:
Meant that they should ONLY cause panning. Note that "small" is defined in the viewportThresholdsAndScales